packages/configure-splash-screen/src/types.ts

```diff
@@ -0,0 +1,274 @@
+import { Color } from 'color-string';
+import { SplashScreenImageResizeModeType, SplashScreenStatusBarStyleType } from './constants';
+
+/**
+ * Type describing generic (platform-agnostic) Splash Screen configuration.
+ */
+export type GenericSplashScreenConfig = {
+  backgroundColor: Color;
+
+  imagePath?: string;
```

Comment:
to be consistent with the rest of our app.json property names with paths we should maybe remove the `Path` suffix - so just `image`.

eg: icon (https://docs.expo.io/versions/latest/config/app/#icon) and image on the current splash config (https://docs.expo.io/versions/latest/config/app/#splash)